docs/source/devel.rst

```diff
@@ -35,6 +37,30 @@ Tips
 * If a test case exceeds many lines consider creating new keywords.
 * When writing tests, pay attention to the number of spaces.
 
+Naming conventions
+------------------
+
+Robot Framework does not strictly enforce naming conventions for suite names and
+test case names, but for this project we will follow consistent and descriptive
+naming conventions for both to improve readability, maintainabilty, and collaboration.
+
+Suite names (Scenario)
```

Comment:
A suite is a scenario?  I thought a test case is a scenario and a suite is a collection of test cases.